Mage.Sets/src/mage/cards/t/TillerEngine.java

```diff
@@ -0,0 +1,61 @@
+package mage.cards.t;
+
+import mage.MageInt;
+import mage.abilities.Ability;
+import mage.abilities.Mode;
+import mage.abilities.common.EntersBattlefieldControlledTriggeredAbility;
+import mage.abilities.effects.common.TapTargetEffect;
+import mage.abilities.effects.common.UntapTargetEffect;
+import mage.cards.CardImpl;
+import mage.cards.CardSetInfo;
+import mage.constants.*;
+import mage.filter.FilterPermanent;
+import mage.filter.common.FilterControlledLandPermanent;
+import mage.filter.common.FilterNonlandPermanent;
+import mage.filter.predicate.permanent.TappedPredicate;
+import mage.target.TargetPermanent;
+
+import java.util.UUID;
+
+/**
+ * @author PurpleCrowbar
+ */
+public final class TillerEngine extends CardImpl {
+
+    private static final FilterControlledLandPermanent filter = new FilterControlledLandPermanent("a land");
+    private static final FilterPermanent filter2 = new FilterNonlandPermanent("nonland permanent an opponent controls");
+
+    static {
+        filter.add(TappedPredicate.TAPPED);
+        filter2.add(TargetController.OPPONENT.getControllerPredicate());
+    }
```

Comment:
We have StaticFilters for both of these. `StaticFilters.FILTER_LAND_A` for "a land" and `StaticFilters.FILTER_OPPONENTS_PERMANENT_NON_LAND` for "nonland permanent an opponent controls".

Reply:
Done